src/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>CSS4 Training</title>
```

Comment:
I think it would be better this way: 
<meta name="viewport" content="width=device-width, initial-scale=1">